src/app/(proper_react)/Shell.tsx

```diff
@@ -24,72 +24,74 @@ export const Shell = (props: Props) => {
 
   return (
     <MobileShell session={props.session}>
-      <div className={styles.wrapper}>
-        <nav className={styles.mainMenu}>
-          <Link href="/" className={styles.homeLink}>
-            <Image
-              src={monitorLogo}
-              alt={l10n.getString("main-nav-link-home-label")}
-              width={170}
-            />
-          </Link>
-          <ul>
-            {/* Note: If you add elements here, also add them to <MobileShell>'s navigation */}
-            <li>
-              <PageLink
-                href="/redesign/user/dashboard"
-                activeClassName={styles.isActive}
-              >
-                {l10n.getString("main-nav-link-dashboard-label")}
-              </PageLink>
-            </li>
-            <li>
-              <a
-                href="https://support.mozilla.org/kb/firefox-monitor-faq"
-                title={l10n.getString("main-nav-link-faq-tooltip")}
-              >
-                {l10n.getString("main-nav-link-faq-label")}
-              </a>
-            </li>
-          </ul>
-        </nav>
-        <div className={styles.content}>
-          <div className={styles.page}>{props.children}</div>
-          <footer className={styles.footer}>
-            <a
-              href="https://www.mozilla.org"
-              className={styles.mozillaLink}
-              target="_blank"
-            >
+      <main>
+        <div className={styles.wrapper}>
+          <nav className={styles.mainMenu}>
+            <Link href="/" className={styles.homeLink}>
               <Image
-                src={mozillaLogo}
-                width={100}
-                alt={l10n.getString("mozilla")}
+                src={monitorLogo}
+                alt={l10n.getString("main-nav-link-home-label")}
+                width={170}
               />
-            </a>
-            <ul className={styles.externalLinks}>
+            </Link>
+            <ul>
+              {/* Note: If you add elements here, also add them to <MobileShell>'s navigation */}
               <li>
-                <a
-                  href="https://support.mozilla.org/kb/firefox-monitor-faq"
-                  title={l10n.getString("footer-external-link-faq-tooltip")}
+                <PageLink
+                  href="/redesign/user/dashboard"
```

Comment:
Q: is "/redesign/" part of the final paths?

Only found 2 references, possibly both from this PR, but it feels like a very temporary path:
```sh
git grep -n "/redesign/" | cat

src/app/(proper_react)/MobileShell.tsx:71:                  href="/redesign/user/dashboard"
src/app/(proper_react)/Shell.tsx:41:                  href="/redesign/user/dashboard"
```

Reply:
It is a temporary path, but it's to avoid conflicting with the existing paths, some of which we will replace. We'll remove that path segment when we go live with this.